test/resource.test.js

```diff
@@ -0,0 +1,373 @@
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /Resource Account Bad Request Account", (done) =>{
+        chai.request(server).post("/resources/account")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idAccount', '6164db5823242f430c487fcb')
+        .attach('file', `${__dirname}/resources/imageAccount.png`, 'imageAccount.png')
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La cuenta no existe");
+            done();
+        });
+    });
+
+    it("POST /Resource Account Bad Request File", (done) =>{
+        chai.request(server).post("/resources/account")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idAccount', '6164db5823242f430c487fca')
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("Necesita subir una imagen");
+            done();
+        });
+    });
+
+    it("POST /Resource Account Bad Request File Extension", (done) =>{
+        chai.request(server).post("/resources/account")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idAccount', '6164db5823242f430c487fca')
+        .attach('file', `${__dirname}/resources/imageError.bmp`, 'imageError.bmp')
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("Extensión no admitida o el archivo debe pesar menos de 10MB");
+            done();
+        });
+    });
+
+    it("POST /Resource Account", (done) =>{
+        chai.request(server).post("/resources/account")
+        .field('Content-Type', 'multipart/form-data')
+        .field('idAccount', '6164db5823242f430c487fca')
+        .attach('file', `${__dirname}/resources/imageAccount.png`, 'imageAccount.png')
+        .end( (error, response) => {
+            response.should.have.status(201);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La imagen se guardo exitosamente");
+            done();
+        });
+    });
+});
+
+describe("POST Tests Resource Lesson",()=>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("POST /Resource Lesson Bad Request", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8292860)
```suggestion
    it("POST /Resource Lesson Bad Request", (done) => {
```